doc/developer/deprecations.rst

```diff
@@ -69,3 +65,7 @@ Version 3.5
 * Remove ``all_triplets`` from ``algorithms/triads.py``
 * Remove ``random_triad`` from ``algorithms/triad.py``.
 * Add `not_implemented_for("multigraph”)` decorator to ``k_core``, ``k_shell``, ``k_crust`` and ``k_corona`` functions.
+* Change ``single_target_shortest_path_length`` in ``algorithms/shortest_path/unweighted.py``
+  to return a dict. See #6527
+* Change ``shortest_path`` in ``algorithms/shortest_path/generic.py``
+  to return a iterator. See #6527
```

Comment:
This is in the version 3.4 header, this should be in 3.5 right?

Reply:
That's just github playing a trick with the collapsible bar - they're under the 3.5 header :upside_down_face:

Reply:
lol